src/Qir/Runtime/lib/QIR/QubitManager.cpp

```diff
@@ -34,54 +34,54 @@ static void FailIf(bool condition, const char* message)
 CQubitManager::QubitListInSharedArray::QubitListInSharedArray(
     QubitIdType startId,
     QubitIdType endId,
-    QubitIdType* sharedQubitStatusArray):
+    QubitIdType* sharedQbitStatusArray):
```

Comment:
I believe the name of this parameter has a typo. Should it be `sharedQubitStatusArray`?

Reply:
Not quite, this is intentional.

The parameter `sharedQubitStatusArray` was shadowing the member variable with the same name in the outer class `CQubitManager`, thus creating a risk of confusing one with the other. And the compiler was warning about that. So I renamed.

Reply:
Ok, thanks for providing more context Robin! If that's the case then it might be worth it to change the name of this variable slightly more (e.g. `sharedQubitStatus`) because maybe one character could still be confusing to the reader. I don't feel strongly about it so I am ok with leaving it in its current state too.

Reply:
I have just noticed this. I'll address in the subsequent PRs.